x-pack/plugins/reporting/index.js

```diff
@@ -156,12 +156,9 @@ export const reporting = (kibana) => {
         xpackMainPlugin.info.feature(this.id).registerLicenseCheckResultsGenerator(checkLicense);
       });
 
-      // Register a function to with Monitoring to manage the collection of usage stats
-      monitoringPlugin && monitoringPlugin.status.once('green', () => {
-        if (monitoringPlugin.collectorSet) {
-          monitoringPlugin.collectorSet.register(getReportingUsageCollector(server));
-        }
-      });
+      // Register a function with server to manage the collection of usage stats
+      const { collectorSet } = server.usage;
```

Comment:
This is a nitpick, but why not just do

```js
// Register a function with server to manage the collection of usage stats
server.usage.collectorSet.register(getReportingUsageCollector(server));
```

Bringing the variable into scope seems unnecessary.